packages/cardano-graphql-services/src/Http/HttpServer.ts

```diff
@@ -1,27 +1,66 @@
+import { Logger, dummyLogger } from 'ts-log';
+import { ProviderError } from '@cardano-sdk/core';
 import { RunnableModule } from '../RunnableModule';
-import { dummyLogger } from 'ts-log';
 import { listenPromise, serverClosePromise } from '../util';
+import bodyParser from 'body-parser';
 import express from 'express';
+import expressPromBundle from 'express-prom-bundle';
 import http from 'http';
 import net from 'net';
+import promClient from 'prom-client';
 
 export type HttpServerConfig = {
-  router: express.Router;
-  name: string;
+  metrics?: {
+    enabled: boolean;
+    options?: expressPromBundle.Opts;
+  };
+  name?: string;
   listen: net.ListenOptions;
 };
 
+export interface HttpServerDependencies {
+  healthCheckCallback: () => Promise<{ ok: boolean }>;
```

Comment:
Exporting `{ok: boolean}` from core Provider.ts would be nice. It's likely to change in the future.
Also, I think a nicer name here is just 'healthCheck' or 'checkHealth' or 'healthCheckImpl'.

I would make it optional, having default just resolve {ok: true}

Reply:
Thanks, I was meaning to come back and define this type in the core. Also agree with the function rename